docker/docker-compose.yaml

```diff
@@ -43,7 +43,7 @@ services:
       - config:/var/huntsman/huntsman-pocs/conf_files
       - panlog:/var/huntsman/logs
       - images:/var/huntsman/images
-    command: [ "panoptes-config-server --verbose run --config-file /var/huntsman/huntsman-pocs/conf_files/huntsman.yaml" ]
+    command: [ "panoptes-config-server --verbose run --no-save-local --no-load-local --config-file /var/huntsman/huntsman-pocs/conf_files/huntsman.yaml" ]
```

Comment:
During our meeting yesterday we were confused as to the path for the config file and it's because we're mapping `huntsman-config` dir to `huntsman-pocs/conf_files/` inside the container.

Not at all related to this PR but thought I would mention here since we're looking at that line.